pages/common/clang-format.md

```diff
@@ -0,0 +1,24 @@
+# clang-format
+
+> A tool to format C/C++/Java/JavaScript/Objective-C/Protobuf/C# code.
+> More information: <https://clang.llvm.org/docs/ClangFormat.html>.
+
+- Format a file and print the result to `stdout`:
+
+`clang-format {{path/to/file}}`
+
+- Format a file inplace:
+
+`clang-format -i {{path/to/file}}`
+
+- Format a file using a predefined coding style:
+
+`clang-format --style={{LLVM|Google|Chromium|Mozilla|WebKit}} {{path/to/file}}`
+
+- Format a file using the `.clang-format` file in the local path:
```

Comment:
Do you mean the current working directory here?

Reply:
From the docs: 
> Use -style=file to load style configuration .clang-format file located in one of the parent directories of the source file
I'm not sure how it's best to phrase it

Reply:
I see. What about this?
```suggestion
- Format a file using the `.clang-format` file in one of the parent directories of the source file:
```